src/vm/exceptionhandling.cpp

```diff
@@ -5288,6 +5281,9 @@ BOOL HandleHardwareException(PAL_SEHException* ex)
         Thread *pThread = GetThread();
         if (pThread != NULL && g_pDebugInterface != NULL)
         {
+#ifdef _TARGET_ARM_
+            HandleArmSingleStep(ex->GetContextRecord(), ex->GetExceptionRecord(), GetThread());
```

Comment:
Last nit - could you please pass in the pThread instead of calling the GetThread again? The GetThread accesses TLS and so passing in the variable that we already  have is cheaper.